src/SearchResults.js

```diff
@@ -0,0 +1,41 @@
+import moment from 'moment';
+import SearchRow from './SearchRow';
+
+function SearchResults(props) {
+  
+  let tr = props.results.map((element) => (
```

Comment:
We are using `let` here, can we think of something better 🤔

Reply:
using const now